lib/rules/name-author-contents.json

```diff
@@ -0,0 +1,17 @@
+{
+	"id": "name-author-contents",
+	"selector": "[role=\"treeitem\"]",
+	"tags": [],
+	"metadata": {
+		"description": "Ensures elements with ARIA roles that require an accessible name have one from author or contents.",
```

Comment:
I would avoid talking about author or content here for the same reasons as stated above.